scripts/ci-build-azure-ccm.sh

```diff
@@ -41,6 +38,8 @@ export CCM_IMAGE_NAME=azure-cloud-controller-manager
 export CNM_IMAGE_NAME=azure-cloud-node-manager
 # cloud node manager windows image version
 export WINDOWS_IMAGE_VERSION=1809
+# container name
+export CONTAINER_NAME=cloud-provider-azure
```

Comment:
Can we name this something besides CONTAINER_NAME?
Maybe AZ_STORAGE_CONTAINTER_NAME?

Reply:
Changed it to `AZURE_BLOB_CONTAINER_NAME`